u1-request-response/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "u1-request-response",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",  "scripts": {
+    "dev": "node app.js -e js,html",
```

Comment:
นี่ควรเป็น 'nodemon' ไม่ใช่ 'node' เราสร้างสคริปต์เพื่อให้เรียกใช้ได้ง่ายขึ้น ("โหนด" ใช้งานได้ค่อนข้างง่ายอยู่แล้ว)

This should say `nodemon`, not `node`. We made the script to make it easier to run it (`node` is already quite easy to run).